src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+$font-color-primary: #060b35;
+$font-color-collateral: #616070;
+$background-color:#fff;
+$border-color: #f3f3f3;
+$button-color:#00acdc;
+$hug: 16px;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: $hug;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    display: block;
+    color: $font-color-primary;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    display: block;
+    color: $font-color-collateral;
+    margin-top: -6px;
+  }
```

Comment:
1. `display: block` is default for `<p>` element
2. Not sure about negative `margin-top`

Reply:
that used to be a code-leftover I didn't notice👍